.env.example

```diff
@@ -1,5 +1,4 @@
 SHOPIFY_API_KEY="YOUR_SHOPIFY_API_KEY"
 SHOPIFY_API_SECRET="YOUR_SHOPIFY_SECRET"
 HOST="YOUR_TUNNEL_URL"
-SHOP="my-shop-name.myshopify.com"
```

Comment:
I think this is used by the CLI - does something break if we keep it?

Reply:
Nothing will break, I just didn't see any usage within the project. I wasn't aware of the CLI using it, restoring this! 🙂